### ISSUE TITLE
Create a reference CobaltCore.dll assembly for use in Github actions

### ISSUE THREAD

**Issue by Shockah** [Shockah/Nickel#51]

**Comment by Fayti1703** [comment]
See branch [`fayti1703/reference-assembly`](https://github.com/Shockah/Nickel/tree/fayti1703/reference-assembly).

Current status:
* We have a `.csproj` to create a reference assembly from a CobaltCore install. (Though this is slightly hacky, I'd *love* to not pull in the entire C# build config and instead just the stuff for `PackageReference` and running MSBuild tasks)
* We have a pre-built reference assembly in Git LFS storage. GitHub has [install instructions for LFS ](https://docs.github.com/en/repositories/working-with-files/managing-large-files/installing-git-large-file-storage) -- might be worth adding those to the README (along with instructions on how to rebuild the assembly, which is just 'run this target in `CobaltCoreReference`').

**Comment by Shockah** [comment]
> The reference assembly is currently not used in building Nickel.

I think that's a good thing, actually. It could be potentially used as a *fallback*, but being able to jump the concrete definitions of CobaltCore members is too useful to just switch altogether.

**Comment by Shockah** [comment]
I guess it actually *has* to be the fallback for it to work with Github actions.

**Comment by Fayti1703** [comment]
Yup -- at minimum in CI we'd need to use the stored assembly rather than a live-generated one.
Though that can also be done by setting an environment variable and doing MSBuild-conditionals based on that.

**Comment by Shockah** [comment]
A bit late update, but: A reference assembly NuGet package has been created, so the task is now partially done: https://www.nuget.org/packages/Nickel.ModBuildConfig.ReferenceAssembly

**Comment by Shockah** [comment]
There now also exist publish scripts, which *can* prepare a Nickel release built against the reference assembly, instead of a real one.
